keras_cv/models/backbones/resnet_v2/resnet_v2_backbone_presets.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""ResNetV2 model preset configurations."""
+
+backbone_presets_no_weights = {
+    "resnet18_v2": {
+        "metadata": {
+            "description": (
+                "ResNet model with 18 layers where the batch normalization "
+                "and ReLU activation precede the convolution layers."
```

Comment:
Is there another option than batch norm and relu precede convolution? Seems weird to comment on this but then all models have this description.

Reply:
Yes, the v1 model has the opposite ordering. Added clarification.